apps/E2E/src/Text/specs/Text.spec.win.ts

```diff
@@ -21,24 +25,50 @@ describe('Text Testing Initialization', function () {
   });
 });
 
-describe('Text Accessibility Testing', () => {
+describe('V1 Text Accessibility Testing', () => {
   beforeEach(async () => {
-    await TextPageObject.scrollToTestElement();
-    await TextPageObject.waitForPrimaryElementDisplayed(PAGE_TIMEOUT);
+    await TextPageObject.scrollToTestElement(await TextPageObject._v1FirstComponent);
+    await TextPageObject.waitForElementDisplayed(await TextPageObject._v1FirstComponent, PAGE_TIMEOUT);
   });
```

Comment:
@samuelfreiberg in another comment you said:

>Or, on second thought, we'd want to scroll to the component lowest on the test page, so probably TextComponentSelector.Fourth

Do we need to scroll to the lowest item on the test page before each test? The tests have been working fine w/o it. Is this just to ensure we're displaying all the test elements before we run a test? If we're really trying to see if the UI element we're testing is displayed before testing it, shouldn't we be searching for that specific UI element instead of the first/last element to be displayed? Or are we just trying to scroll all the test elements into view for screenshot/errorshots purposes?

Also, shouldn't calling `scrollToTestElement()` be enough here since it checks if the specified Element is displayed?